examples/planning/generate_forecast_metrics.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example generates forecast metrics for a keyword plan.
+
+To create a keyword plan, run the add_keyword_plan.py example.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, keyword_plan_id):
+    keyword_plan_service = client.get_service('KeywordPlanService')
+    resource_name = keyword_plan_service.keyword_plan_path(customer_id,
+                                                           keyword_plan_id)
+
+    try:
+        response = keyword_plan_service.generate_forecast_metrics(resource_name)
+    except GoogleAdsException as ex:
+        print('Request with ID "%s" failed with status "%s" and includes the '
+              'following errors:' % (ex.request_id, ex.error.code().name))
+        for error in ex.failure.errors:
+            print('\tError with message "%s".' % error.message)
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print('\t\tOn field: %s' % field_path_element.field_name)
+        sys.exit(1)
+
+    for i, forecast in enumerate(response.keyword_forecasts):
+        print('#%s Keyword ID: %s' % (i + 1,
```

Comment:
what's with the mysterious i+1?

Reply:
These come back in a list so this just prints the number of the keyword, taken from other examples [here](https://github.com/googleads/google-ads-dotnet/pull/34/files#diff-d7bc57cd34475a83a64361d1e5a68011R78) and [here](https://github.com/googleads/google-ads-ruby/pull/64/files#diff-96b31380f25191881a16c6d1bf076d1aR36)